voicevox_engine/user_dict.py

```diff
@@ -99,7 +101,7 @@ def update_dict(
         raise RuntimeError("辞書のコンパイル時にエラーが発生しました。")
     pyopenjtalk.unset_user_dict()
     try:
-        tmp_dict_path.replace(compiled_dict_path)
+        shutil.move(tmp_dict_path, compiled_dict_path)
```

Comment:
shutil.moveを使わないといけないということがわかるように、コメントを残しておくのはどうでしょう？👀
```suggestion
        shutil.move(tmp_dict_path, compiled_dict_path) # ドライブを跨ぐためPath.replaceが使えない
```